commands/serverless.go

```diff
@@ -359,21 +365,36 @@ func showLanguageInfo(c *CmdConfig, APIHost string) error {
 func RunServerlessUndeploy(c *CmdConfig) error {
 	haveArgs := len(c.Args) > 0
 	pkgFlag, _ := c.Doit.GetBool(c.NS, "packages")
+	trigFlag, _ := c.Doit.GetBool(c.NS, "triggers")
 	all, _ := c.Doit.GetBool(c.NS, "all")
 	if haveArgs && all {
 		return errUndeployAllAndArgs
 	}
 	if !haveArgs && !all {
 		return errUndeployTooFewArgs
 	}
+	if pkgFlag && trigFlag {
+		return errUndeployTrigPkg
+	}
+	if all && trigFlag {
+		return cleanTriggers(c)
+	}
 	if all {
 		return cleanNamespace(c)
 	}
 	var lastError error
 	errorCount := 0
+	var ctx context.Context
+	var sls do.ServerlessService
+	if trigFlag {
+		ctx = context.TODO()
+		sls = c.Serverless()
+	}
 	for _, arg := range c.Args {
 		var err error
-		if strings.Contains(arg, "/") || !pkgFlag {
+		if trigFlag {
+			err = sls.DeleteTrigger(ctx, arg)
```

Comment:
Undeploy is currently a bit irregular in that it continues to rely on the plugin for _most_ cases but the deployer has no direct analog for deleting just a trigger without its attached function.  So, here, we call the triggers API directly to effect deletion.  This will all get simplified / consolidated as part of the ongoing effort to reduce dependence on the plugin.